proto/build.rs

```diff
@@ -84,6 +84,7 @@ static TYPE_ATTRIBUTES: &[(&str, &str)] = &[
     (".penumbra.crypto.Value", SERIALIZE),
     (".penumbra.crypto.Denom", SERIALIZE),
     (".penumbra.crypto.Denom", SERDE_TRANSPARENT),
+    (".penumbra.chain.ChainState", SERIALIZE),
```

Comment:
Is this supposed to be `ChainParams`? I don't think we have a `ChainState`.  If it is supposed to be `ChainParams`, actually using the serialize impl requires an extra annotation, as in #363

Reply:
yes, it should be, but now it appears to be unused? deleting for now.